src/main/java/de/tum/in/www1/artemis/repository/CourseRepository.java

```diff
@@ -354,6 +360,32 @@ default Course findWithEagerLearningGoalsByIdElseThrow(long courseId) {
         return findWithEagerLearningGoalsById(courseId).orElseThrow(() -> new EntityNotFoundException("Course", courseId));
     }
 
+    /**
+     * Utility method to check if the messaging feature is enabled for a course.
+     * <p>
+     * Keep in sync with the implementation in {@link CourseInformationSharingConfiguration}
```

Comment:
```suggestion
     * Checks if the messaging feature is enabled for a course.
```
The enum no longer has such a method.